tests/Tests/500__jupyterhub/test-jupyterlab-git.robot

```diff
@@ -0,0 +1,162 @@
+*** Settings ***
+Resource         ../../Resources/ODS.robot
+Resource         ../../Resources/Common.robot
+Resource         ../../Resources/Page/ODH/JupyterHub/JupyterHubSpawner.robot
+Resource         ../../Resources/Page/ODH/JupyterHub/JupyterLabLauncher.robot
+Library          OpenShiftCLI
+Library          DebugLibrary
+Suite Setup      Server Setup
+Suite Teardown   End Web Test
+
+
+*** Variables ***
+${REPO_URL} =    *****
+${DIR_NAME} =  Python
+${FILE_PATH} =   Python/file.ipynb
+${EMAIL_ID} =  user@gmail.com
+${NAME} =  user
+${COMMIT_MSG} =  commit msg2
+
+
+*** Test Cases ***
+
+Verify Pushing Project Changes Remote Repository
+    [Tags]  ODS-326
+    ...     Tier1
+    ${randnum}=  Generate Random String  9  [NUMBERS]
+    ${commit_message} =    Catenate    ${COMMIT_MSG}    ${randnum}
+    Push Some Changes to Repo    ${GITHUB_USER.USERNAME}  ${GITHUB_USER.TOKEN}   ${FILE_PATH}    ${REPO_URL}    ${commit_message}
+
+Verify updating your project with changes from a git repository
+  [Tags]    ODS-324
+  ...       Tier1
+  Clone Git Repository And Open    ${REPO_URL}    ${FILE_PATH}
```

Comment:
Try to keep the indentation the same across TCs

Reply:
okay